### DISCUSSION TITLE
Defining loss function in integral form

### DISCUSSION THREAD

**Discussion by ali-fallah-66** [lululxvi/deepxde#1537] [Q&A]
Hi, I hope you are fine. I have a question and would be glad if you help me with it. Is possible to define the loss function in integral form over the whole domain, for example  

loss= int  (0 to 1) (dy/dx+ y)  
 
where y=y(x) is the unknown function to be determined. 

Best

**Comment by lululxvi** [comment]
See the code in the paper https://doi.org/10.1137/21M1397908
